packages/core/package.json

```diff
@@ -119,7 +119,7 @@
     "react-transition-group": "^4.4.5",
     "react-virtualized-auto-sizer": "^1.0.7",
     "react-window": "^1.8.7",
-    "react-windowed-select": "^2.0.4",
+    "react-windowed-select": "3.1.2",
```

Comment:
in 3.1.1 react windowed select added typescript types, that's why i migrated it
<img width="384" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/123493087/57d4dce3-be6d-46d4-87cd-b7a7d10d057c">

Reply:
There's a major bump in react-select in this library between those two versions (see [this](https://github.com/jacobworrel/react-windowed-select/compare/2.0.3-beta.0...ef0ed2e43d989347a1ef7d9bb4219e7247517cd0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R28)), this might lead to issues as it's a depenedency